subsys/net/ip/ipv6.c

```diff
@@ -2123,11 +2118,12 @@ int net_ipv6_send_ns(struct net_if *iface,
 	net_icmpv6_set_ns_hdr(pkt, ns_hdr);
 
 	if (is_my_address) {
+		u16_t *len = &NET_IPV6_HDR(pkt)->len;
```

Comment:
I'd say, this gets hairy and hardly an improvement. Please use standard var arithmetics, without pointers.

Reply:
As for my taste, the original snippet is slightly overloaded visually and using pointer here makes it easier to parse what's going on. But, don't really want on to argue on such matters of taste, OK, corrected.